lib/workos/sso/sso.ex

```diff
@@ -55,31 +79,39 @@ defmodule WorkOS.SSO do
   end
 
   def get_authorization_url(_params, _opts),
-    do: raise(ArgumentError, message: "Either domain, provider, or connection required in params")
+    do: raise(ArgumentError, message: "Either connection, domain, provider, or organization required in params")
 
   @doc """
-  Create a Connection
+  Fetch the user profile details with an access token.
 
   ### Parameters
-  source (source) The Draft Connection token that's been provided to you by WorkOS.js
+  - access_token (string) An access token that can be exchanged for a user profile
 
   ### Example
-  WorkOS.SSO.create_connection('draft_conn_12345')
+  WorkOS.SSO.get_profile("12345")
   """
-  def create_connection(source, opts \\ []) do
-    post("/connections", %{source: source}, opts)
+  def get_profile(access_token, opts \\ []) do
```

Comment:
This function redefinition is a breaking change that broke the existing usage. :(

Reply:
@seantanly you're correct, this is a breaking change. We updated this here to better align with our [API endpoints for retrieving a profile](https://workos.com/docs/reference/sso/profile). The previous `get_profile` function has now been renamed to `get_profile_and_token`, so you may need to update your code to reflect that change as well. While we're hoping to limit the amount of breaking changes introduced, this SDK is currently considered experimental, and in this case we felt it was warranted.